lighthouse-core/config/default-config.js

```diff
@@ -479,7 +479,7 @@ const defaultConfig = {
         {id: 'canonical', weight: 1, group: 'seo-content'},
         {id: 'font-size', weight: 1, group: 'seo-mobile'},
         {id: 'plugins', weight: 1, group: 'seo-content'},
-        {id: 'structured-data-automatic', weight: 1, group: 'seo-content'},
+        // {id: 'structured-data-automatic', weight: 1, group: 'seo-content'},
```

Comment:
Maybe there's a better way to disable the audit for now than what I'm doing in [this commit](https://github.com/GoogleChrome/lighthouse/pull/5446/commits/74de3a252dd6160b4339fb0732aa0be2dfe6bdce)?